Formula/mercurial.rb

```diff
@@ -13,6 +14,14 @@ class Mercurial < Formula
     sha256 "8934a8fc26c36e56f017d99ac43a0ebe9d6922f8276c1dfccd9ccb980ae05e45" => :mavericks
   end
 
+  option "with-custom-python", "Install against the python in PATH instead of "\
```

Comment:
i like the continuation better than a heredoc :p but this basically fits on a line; if it runs longer than 80 characters will anyone be mad at me